clusterloader2/testing/experiments/enable_restart_count_check.yaml

```diff
@@ -0,0 +1,2 @@
+ENABLE_RESTART_COUNT_CHECK: true
+IGNORE_RESTARTS_OF: prometheus-to-sd-exporter,metadata-proxy,kube-proxy,fluentd-gcp,kubernetes-dashboard,kube-scheduler,kube-controller-manager
```

Comment:
I think this list should be per test type, like:
* gce 5000: with disabled fluentd-gcp
* kubemark: kube-scheduler, kube-controller-manager

This way we will be able to detect if kube-scheduler is restarting in gce 5000 (which would be a regression) even before we debug this in kubemark